gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/Oplog.java

```diff
@@ -1170,6 +1170,13 @@ private void closeAndDeleteAfterEx(IOException ex, OplogFile olf) {
   }
   
   private void preblow(OplogFile olf, long maxSize) throws IOException {
+
+    GemFireCacheImpl.StaticSystemCallbacks ssc = GemFireCacheImpl.getInternalProductCallbacks();
+    if(ssc != null && ssc.isSnappyStore() && ssc.isAccessor()){
+      logger.warning(LocalizedStrings.SHOULDNT_INVOKE, "Pre blow is invoked on Accessor Node.");
```

Comment:
Oplog.preblow is never called on lead node as createCrf and createDrf methods are not called in case of lead node. 
Therefore I did not see this warning on a lead at all.